examples/platforms/efr32mg12/radio.c

```diff
@@ -516,7 +516,7 @@ otError otPlatRadioTransmit(otInstance *aInstance, otRadioFrame *aFrame)
         sCurrentBandConfig = config;
     }
 
-    RAIL_WriteTxFifo(gRailHandle, &aFrame->mLength, sizeof(aFrame->mLength), true);
+    RAIL_WriteTxFifo(gRailHandle, (uint8_t *)&aFrame->mLength, sizeof(aFrame->mLength), true);
```

Comment:
The frame mLength field has been enlarged to a uint16_t from a uint8_t.  Later commit 7f72f7d9 fixes this by converting it to a uint8_t prior to writing it to the Tx Fifo.